.ci/install_runtime.sh

```diff
@@ -63,6 +62,17 @@ build_install_shim_v2(){
 
 build_install_shim_v2
 
+build_install_agent_ctl(){
+	bash "${cidir}/install_rust.sh" && source "$HOME/.cargo/env"
```

Comment:
AFAIR we're not building any rust component on the host side, but rather building them inside a container.
Would be possible to do the same here?

Reply:
I'm not super familiar with the project still, and I'm a bit confused. the agent-ctl runs on the host side, right?

Reply:
@bookinabox, oops, my bad, let me take a step back here.

It runs on the host side, but what I was afraid of was this being the only reason we end up installing rust in the CI machines, just to build the `agent-ctl` tool.  If that was the case, I'd suggest we rather build it inside a container (which would have rust installed there) and just install the binary in the correct location.

However, it seems we're already installing rust on the Ci machine as we build `virtiofsd` there (for non-x86_64 machines), thus I think this one is fine to be merged as it's.

Does the explanation make sense?

Reply:
yes, thanks!